packages/components/src/styles/components/accordion.scss

```diff
@@ -51,6 +52,7 @@
   --hds-accordion-item-content-padding-left: 12px;
   --hds-accordion-item-gap: 12px;
   --hds-accordion-item-icon-size: 20px;
+  --hds-accordion-item-border-radius: 6px;
```

Comment:
Should this one be 5px rather than 6px?

Reply:
This doesn't currently reflect what I proposed above, but rather reflects what is currently in Figma.

Reply:
Ah I see, I misread the description.